cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -74,6 +75,7 @@ type StaticAutoscaler struct {
 	processorCallbacks      *staticAutoscalerProcessorCallbacks
 	initialized             bool
 	ignoredTaints           taints.TaintKeySet
+	utilizationTracker      *utilization.Tracker
```

Comment:
Discussed offline with @x13n. I think we agree that it would be beneficial, for example, in order to produce utilization metrics. It's not in the scope of this PR though and can be done as a separate change later on.